src/services/Dashboard/index.js

```diff
@@ -67,7 +67,6 @@ export const SERVICES = {
 
     getHeatmapTiles(fromDate, toDate, zoomLevel, maintopic, tileid, periodType, 
                     dataSource, externalsourceid, conjunctivetopics, callback) {
-        console.log(`processing tile request [${maintopic}, ${fromDate}, ${toDate}, ${tileid}}]`)
```

Comment:
can we leave this in for the moment as I depend on this for monitoring / debugging. this will be removed before we go live.

Reply:
This creates a lot of noise and makes other debugging harder... but sure.  84969b0